game/world/objects/npc.cpp

```diff
@@ -257,7 +257,7 @@ void Npc::load(Serialize &fin, size_t id) {
   loadAiState(fin);
 
   fin.read(currentInteract,currentOther,currentVictum);
-  fin.read(currentLookAt,currentTarget,nearestEnemy);
+  fin.read(currentLookAtNpc,currentTarget,nearestEnemy);
```

Comment:
1. `write` code wasn't updated to mirror read
2. Waypoint `currentLookAt` is not serialized 

For proper serialization, you need to bump save file version in `Serialize::Version` and add check for `fin.version()` before reading new data.

Reply:
I'm not exactly sure how the serialization works. But I think it's fixed now